clang/test/CAS/debug_info_split.c

```diff
@@ -2,6 +2,12 @@
 // RUN:   -fcas-friendly-debug-info=debug-abbrev -Xclang -fcas-backend \
 // RUN:   -Xclang -fcas-backend-mode=verify
 
+// RUN: %clang -g -c -o %t.casid %s --target=arm64-apple-darwin10 \
+// RUN:   -fcas-friendly-debug-info=debug-abbrev -Xclang -fcas-backend \
+// RUN:   -Xclang -fcas-backend-mode=casid \
+// RUN:   -Xclang -fcas-path -Xclang %t.cas
+// RUN: llvm-cas-mc-dump --cas=%t.cas --casid-file %t.casid --dwarf-sections-only | FileCheck %s
+
```

Comment:
The primary test for an LLVM tool shouldn't be in clang. Could we create LLVM IR out of this C file (explain in a comment how it was generated) and move the test to `llvm/test/CAS`?

Reply:
Yeah, I wasn't too happy about this. Let me give the IR idea a try